recipes-alexa/apl-client-library/apl-client-library.inc

```diff
@@ -0,0 +1,37 @@
+SUMMARY = "APL Client Library"
+DESCRIPTION = "Recipe to build APL Client Library"
+LICENSE = "Apache-2.0"
+LIC_FILES_CHKSUM = "file://LICENSE;md5=3b83ef96387f14655fc854ddc3c6bd57"
+
+FILESEXTRAPATHS_prepend:= "${THISDIR}/files:"
+
+OECMAKE_GENERATOR = "Unix Makefiles"
+
+SRC_URI = "git://github.com/alexa/apl-client-library.git;protocol=https;nobranch=1"
+SRCREV ?= "v${PV}"
+S = "${WORKDIR}/git"
+
+inherit pkgconfig cmake
+
+DEPENDS = "alsa-utils apl-core-library asio websocketpp"
+
+EXTRA_OECMAKE = " \
```

Comment:
Can you please append instead of assigning?

Reply:
done